### PR TITLE
Feature/one timeline

### PR DESCRIPTION
**Pull request by h164654156465**
Prior to these changes, timeline data are stored both in home window and controlbar window. Those data are constantly being synced to each other, which causes performance issues as well as other unpredicable bugs.
Thus, I leave only one timeline data(stored in state) in home window, and allow users to click on coontrolbar window to interact with the original functionality that controlbar window had.

### CODE REVIEW

File: src/container/BlockContainer.jsx [hunk 1]
```diff
@@ -34,17 +41,20 @@ export class BlockContainer extends Component {
...
-        ipcRenderer.on('cb-sync-with-slu', (event, args) => {
-            ipcRenderer.send('init-slu-title', args.slu.name);
-            this.setState({
-                sluPath: args.sluPath,
-                slu: args.slu
+        ipcRenderer.once('init-tl', (event, args) => {
+            jsonManager.readJSON(args.path).then((slu) => {
+                ipcRenderer.send('init-tl-title', slu.name);
+                this.setState({
+                    slu: slu,
```
Comment (niennnnnnn) [inline src/container/BlockContainer.jsx]: 為什麼slu還在，可以把它換成現在的名字嗎><

Comment (h164654156465) [reply src/container/BlockContainer.jsx]: 阿對齁，趁這次改動順便把名字改一改嗎?

Comment (niennnnnnn) [reply src/container/BlockContainer.jsx]: 當然啦

Comment (h164654156465) [reply src/container/BlockContainer.jsx]: 那我統一改成 tl(timeline)

File: src/renderer/video-recorder.js [hunk 2]
```diff
@@ -7,12 +7,14 @@ const path = require('path');
 const uuidv1 = require('uuid/v1');
 
 const userPath = app.getPath('userData').replace(/\\/g, '\\\\');
-let recorder;
+export let videoRecorder;
```
Comment (LOTINGYI) [inline src/renderer/video-recorder.js]: 要不要把這行export還有下面的export合併成
module.export = {
    videoRecorder,
    videoRecordStart: function(){},
    videoRecordStop: function(){}
}

Comment (h164654156465) [reply src/renderer/video-recorder.js]: got it. Sounds like a great idea.

File: src/container/BlockContainer.jsx (hunk 1 above)
Comment (h164654156465) [reply src/container/BlockContainer.jsx]: 下一個 feature branch 我會調整檔案結構，到時候一起再改動

**Comment by niennnnnnn** [comment]
ok

**Pull request merged**